Kudu.Core/Deployment/Generator/FunctionNodeBuilder.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Kudu.Contracts.Settings;
+
+namespace Kudu.Core.Deployment.Generator
+{
+    class FunctionNodeBuilder : BaseBasicBuilder
```

Comment:
It's not really about Node, but about it not being compiled. e.g. if you use C# with .csx file, or PowerShell functions, you're still in that case